src/main/webapp/user-private-comments.jsp

```diff
@@ -14,27 +13,56 @@
 %>
 
 <%--HTML--%>
-<ul>
-  <%if (currUser) {%>
-    <h1 id="pc-header">Your Private Task Comments:</h1>
-    <%for (TaskData task : tasks) {%>
-      <li class="task-with-pc">
-        <button class="inline deep-button" type="button" onclick="location.href='task?taskID=<%=task.getTaskID()%>'">
-          <%=task.getName()%>
-        </button>
-        <%request.setAttribute("task", task);%>
-        <%request.setAttribute("clickable", false);%>
-        <jsp:include page="task-status-checkmark.jsp"/>
-        <p class="inline"><%=task.getDescription()%></p>
-        <form action="/user-profile" method="POST">
-          <input type="hidden" name="taskID" value="<%=task.getTaskID()%>">
-          <br>
-          <textarea style="width:300px; height:100px;" type="text" name="message-<%=task.getTaskID()%>" rows="6" cols="50"><%=(privateCommentsMap.get(task.getTaskID())).getMessage().trim()%></textarea>
-          <br>
-          <button class="deep-button" style="font-size: 12px;" type="submit" class="deep-button">Update Comment</button>
-        </form>
-        <br><br>
-      </li>
-    <%}%>
-  <%}%>
-</ul>
\ No newline at end of file
+<html>
+  <head>
+    <script src="https://cdn.jsdelivr.net/npm/@editorjs/editorjs@latest"></script>
+    <script src="https://cdn.jsdelivr.net/npm/@editorjs/header@latest"></script>
+  </head>
+  <script>
+    function init(privateCommentID, jsData) {
+      const editor = new EditorJS({
+        holderId : privateCommentID,
+        autofocus: true,
+        data: jsData,
+        tools: {
+          header: Header,
+        },
+      });
+    }
+    function save() {
+      editor.save().then((outputData) => {
+        console.log('Article data: ', outputData)
+      }).catch((error) => {
+        console.log('Saving failed: ', error)
+      });  
+    }
+  </script> 
+  <body>  
+    <ul>
+      <%if (currUser) {%>
+        <h1 id="pc-header">Your Private Task Comments:</h1>
+        <%for (TaskData task : tasks) {%>
+          <%System.out.println(task);
+          <li class="task-with-pc">
+            <button class="inline deep-button" type="button" onclick="location.href='task?taskID=<%=task.getTaskID()%>'">
+              <%=task.getName()%>
+            </button>
+            <%request.setAttribute("task", task);%>
+            <%request.setAttribute("clickable", false);%>
+            <jsp:include page="task-status-checkmark.jsp"/>
+            <p class="inline"><%=task.getDescription()%></p>
+            <form onload="init(box-<%=task.getTaskID()%>, <%=(privateCommentsMap.get(task.getTaskID())).getMessage().trim());" action="/user-profile" method="POST">
```

Comment:
What out with this, after you get past the jsp error your js will probably error out since box-12345 is not quoted.